terraform/azure/projects/utoronto.tfvars

```
@@ -0,0 +1,19 @@
+subscription_id    = "ead3521a-d994-4a44-a68d-b16e35642d5b"
+resourcegroup_name = "2i2c-utoronto-cluster"
+
+ssh_pub_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQJ4h39UYNi1wybxAH+jCFkNK2aqRcuhDkQSMx0Hak5xkbt3KnT3cOwAgUP1Vt/SjhltSTuxpOHxiAKCRnjwRk60SxKhUNzPHih2nkfYTmBBjmLfdepDPSke/E0VWvTDIEXz/L8vW8aI0QGPXnXyqzEDO9+U1buheBlxB0diFAD3vEp2SqBOw+z7UgrGxXPdP+2b3AV+X6sOtd6uSzpV8Qvdh+QAkd4r7h9JrkFvkrUzNFAGMjlTb0Lz7qAlo4ynjEwzVN2I1i7cVDKgsGz9ZG/8yZfXXx+INr9jYtYogNZ63ajKR/dfjNPovydhuz5zQvQyxpokJNsTqt1CiWEUNj georgiana@georgiana"
+
+global_container_registry_name = "2i2cutorontohubregistry"
+global_storage_account_name = "2i2cutorontohubstorage"
```

```suggestion
global_container_registry_name = "2i2UTorontoHubRegistry"
global_storage_account_name = "2i2cUTorontoHubStorage"
```

Just to follow convention laid out in the "Naming Convention Guidelines for Container Registries and Storage Accounts" notebox on this page (under the Azure tab) https://infrastructure.2i2c.org/en/latest/howto/operate/new-cluster/new-cluster.html

---

@sgibson91, terraform seems to be complaining about using uppercase letters for the storage account names.

```
│ Error: name ("2i2cUTorontoHubStorage") can only consist of lowercase letters and numbers, and must be between 3 and 24 characters long

```